apps/pie-microsite/src/_11ty/shortcodes/list.js

```diff
@@ -0,0 +1,20 @@
+const markdownFilter = require('../filters/markdown');
+
+/**
+ * A List HTML component
+ * @param {string} type - Type of list: orderd, pill
+ * @param {string} items - An array of list items
+ * @returns {string}
+ */
+// eslint-disable-next-line func-names
+module.exports = function ({ type, items }) {
+    const listItems = items.map(item => `<li class="c-list-item"><span>${markdownFilter(item, true)}</span></li>`).join('');
+    const listClasses = [
+        'c-list',
+        `c-list--${type}`
```

Comment:
not critical but this allows the author to inject anything into the css class. We could always create a map of valid types or something and try to retrieve the value they provide from it (so they give the shortcode a key to look up). Then we can provide a default or fail the build if its an invalid type.